lib/mllp/client.ex

```diff
@@ -635,11 +636,11 @@ defmodule MLLP.Client do
   @default_opts %{
     telemetry_module: MLLP.DefaultTelemetry,
     tls_opts: [],
-    socket_opts: []
+    socket_opts: [send_timeout: 60_000]
```

Comment:
send_timeout has two flavors, a timeout in sending will result a disconnection of the conn. This can be disabled, but it's probably the best thing to do, especially with MLLP / HL7.

Reply:
Is this understanding correct:
- MLLP.Client.send/4 timeout which defaults to `infinity` is the `gen_server` call timeout. It's not related to socket send_timeout.
- Socket send_timeout which now defaults to 60_000 can only be provided before starting a client connection.
- reply_timeout which defaults to 60_000 is for receiving ACK which can be provided on each send call using options `%{reply_timeout: 1_000}`

Reply:
That is 100% correct.